clusters/kind-cluster/infra/monitoring/release.yaml

```diff
@@ -1,24 +0,0 @@
----
-apiVersion: helm.toolkit.fluxcd.io/v2beta1
-kind: HelmRelease
-metadata:
-  name: monitoring
-  namespace: dscp
-spec:
-  install:
-    remediation:
-      retries: -1
-  releaseName: prometheus
-  chart:
-    spec:
-      chart: kube-prometheus-stack
-      sourceRef:
-        kind: HelmRepository
-        name: prometheus-community
-  interval: 10m0s
-  # Default values
-  # https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml
-  valuesFrom:
-    - kind: ConfigMap
-      name: monitoring
-      valuesKey: values.yaml
```

Comment:
Removing this will mean the serviceMonitors will fail to be installed and the helm installs will fail, unless the serviceMonitors are disabled across the board in `values.yaml`

Reply:
bit late to worry about this. Helm releases installed successfully so I'm going to assume not an issue for now. We can worry about this if/when we start using the kind cluster again. The main concern of this PR was removing deprecated charts/services and doing the rename